docs/topics/native/native-memory-manager.md

```diff
@@ -54,9 +54,49 @@ build script:
 
 ## Memory consumption
 
-If there are no memory leaks in the program, but you still see unexpectedly high memory consumption, 
+### Check for memory leaks
+
+To access the memory manager metrics, call `kotlin.native.internal.GC.lastGCInfo()`. It returns statistics for the last
+run of the garbage collector. The statistics can be useful for:
+
+* Debugging memory leaks when using global variables
+* Checking if there are any leaks when running tests
+
+```kotlin
+import kotlin.native.internal.*
+import kotlin.test.*
+
+class Resource
+
+val global = mutableListOf<A>()
```

Comment:
probably it should be Resource here, not A

Reply:
Good catch! missed that one